dolphinscheduler-bom/pom.xml

```
@@ -67,6 +67,7 @@
         <guava.version>31.1-jre</guava.version>
         <postgresql.version>42.4.1</postgresql.version>
         <hive-jdbc.version>2.3.9</hive-jdbc.version>
+        <kyuubi-jdbc.version>1.6.0-incubating</kyuubi-jdbc.version>
```

Kyuubi 1.7.0 is out a few weeks ago

```suggestion
        <kyuubi-jdbc.version>1.7.0</kyuubi-jdbc.version>
```

---

OK. Thanks.